testdata/validate/par3stdout.log

```diff
@@ -0,0 +1,25 @@
+InputFile: ../par1.crs
+ConfigFile: bde_copy.cfg
+ConfigFile: ../test3.cfg
+OutputFile: -
+TableName: crs_parcel
+InputFields: id|test_date|img_id|fen_id|toc_code|alt_id|area|nonsurvey_def|appellation_date|parcel_intent|status|total_area|calculated_area|se_row_id|audit_id|shape
+OutputFields: id|test_date|img_id|fen_id|toc_code|alt_id|area|nonsurvey_def|appellation_date|parcel_intent|status|total_area|calculated_area|se_row_id|audit_id|shape
+BdeStartTime: 2009-10-14 23:26:56
+BdeEndTime: 2009-10-15 23:47:45
+BdeSize: 12710
+
+Warning: Date value of column appellation_date is outside valid range
+At record 2 of ../par1.crs
+3007668|01/05/2010|||PRIM|||DCDB Document Id: CT 247/5|1012-10-01 02:33:44|DCDB|CURR|1202928.0||3007668|1917|1 MULTIPOLYGON (((-10.9133899333 -45.0542126167)))|
```

Comment:
mixing logs and output in the same stream looks wrong, shouldn't logs go to stderr when output is stdout and log filename is not given ?

Reply:
maybe we should just FORBID not specifying an explicit log filename when stdout is requested for output... could be cleaner and reduce errors

Reply:
I have changed it so that you must enter a log filename if output is requested to stdout, I have also updated the help to reflect this.